src/DnsRootStack.ts

```diff
@@ -46,6 +61,59 @@ export class DnsRootStack extends cdk.Stack {
 
   }
 
+  setupDnsManagementRole() {
+    const role = new IAM.Role(this, 'role', {
+      roleName: 'dns-manager',
+      description: 'Role for dns-management account with access rights to IAM (readonly) and Route53',
+      assumedBy: new IAM.PrincipalWithConditions(
+        new IAM.AccountPrincipal(Statics.iamAccountId), //IAM account
+        {
+          Bool: {
+            'aws:MultiFactorAuthPresent': true,
+          },
+        },
+      ),
+    });
+
+    /**
+     * Policy for route53
+     */
+    role.addToPolicy(
+      new IAM.PolicyStatement({
+        effect: IAM.Effect.ALLOW,
+        actions: [
+          'route53:*', // Allow domain management
+          'route53domains:*', // Allow domain management
+        ],
+        resources: [
+          '*',
```

Comment:
Wildcard nodig?

Reply:
eens met Martijn; even kritisch kijken of wildcards idd nodig zijn

Reply:
De resources bestaan nog niet, dus arn definieren is niet mogelijk. Ik weet ook niet of het wenselijk is dit te beperken tot bijvoorbeeld de csp-nijmegen.nl hosted zone en domein naam zelf?